naff/models/discord/channel.py

```diff
@@ -1926,11 +1977,11 @@ async def edit(
         Args:
             name: 1-100 character channel name
             archived: whether the thread is archived
-            applied_tags: list of tags to apply to a forum post (!!! This is for forum threads only)
+            applied_tags: list of tags to apply
```

Comment:
After a post is edited with new tags, the cache does not get updated with them
![image](https://user-images.githubusercontent.com/38250010/190852980-36d09dbb-f344-4653-87ef-19a83badb755.png)
![image](https://user-images.githubusercontent.com/38250010/190852987-9a2e15dc-9fa2-4ec7-91ef-5cc2735177ab.png)

Reply:
Interesting. Would have thought gateway would have taken care of that. I'll look into it